ynput/OpenPype
# TrayPublisher: editorial with folder path

## Changelog Description
Support for folder path unique identifiers in editorial and particularly in Traypublisher. This allows multiple shots with the same names in different sequences to be accepted during publishing.

## Additional info
- Since it is based on https://github.com/ynput/OpenPype/pull/5829 branch - this PR is targeted temporarily back to it so we can see lesser changes in files.

## Testing notes:
1. Open Traypublisher and use simple editorial publishing workflow
2. You will need to docker compose with `ynput/ayon:experimental` (let me know if you need help with it).
3. here is example of EDL to use. You can use it with any of yours vide mp4/mov 
[simple_editorial_setup__ayon_folderPath.zip](https://github.com/ynput/OpenPype/files/13307692/simple_editorial_setup__ayon_folderPath.zip)
4. ideally you will also need to change Ayon Resolve addon settings in following way. 
- go to your **Project settings** and into `ayon+settings://traypublisher/editorial_creators/editorial_simple/shot_rename/shot_rename_template?project=<<<	YOUR PROJECT NAME>>>`
- change value of shot renamer to `{_shot_}` token only.
- Then open Traypublisher in the context of the project and ideally `/editorial` asset and task **edit** and set Editorial Simple. 
**Warning!!** - since there is a bug fixed in this https://github.com/ynput/OpenPype/pull/5880, you will need to make sure there is no task and this settings is empty: `ayon+settings://traypublisher/editorial_creators/editorial_simple/shot_add_tasks?project=<<<	YOUR PROJECT NAME>>>`
